wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/DevfileConverter.java

```diff
@@ -105,6 +113,15 @@ public DevfileImpl workspaceToDevFile(WorkspaceConfigImpl wsConfig)
     return devfile;
   }
 
+  @Override
+  public WorkspaceConfig convert(DevfileImpl devfile) throws ServerException {
```

Comment:
Are you planning to add unit tests for this code?

Reply:
Added